### ISSUE TITLE
Bug with container

### ISSUE THREAD

**Issue by morinfa** [FluidTYPO3/flux#1111]
I' ve followed the documentation to create a container in the Configuration section of a FluidContent, but an error occurs and the container is badly displayed.

`Warning: Invalid argument supplied for foreach() in vendor/typo3/cms/typo3/sysext/backend/Classes/Form/Container/FlexFormElementContainer.php on line `48`

Here is the code I use (taken from the documentation) :
`<flux:form.container name="settings.name" label="Name">`
`    <flux:field.input name="firstname" label="First name"/>`
`    <flux:field.input name="lastname" label="Last name"/>`
`</flux:form.container>`

Here is the result in the backend :
![container](https://cloud.githubusercontent.com/assets/4995658/14220804/b19b1e5c-f862-11e5-82f5-ad490fa6bcf2.png)

TYPO3 7.6.4
Flux 7.3.0 (dev branch)
FluidContent 4.4.1

**Comment by morinfa** [comment]
Related to #1105

**Comment by NamelessCoder** [comment]
Duplicate of https://github.com/FluidTYPO3/flux/issues/1086 - summary: core regression, fix awaited in TYPO3 7.6.5.

**Comment by manuelbachl** [comment]
@NamelessCoder Is this really resolved? For me (just updated to 7.6.5 right now) the error still gets thrown...

my (simplyfied) code:

```
<f:section name="Configuration">
    <flux:form
            id="CallToActionBoxes"
            label="Call to action Boxen (nebeneinander)"
            description="Call to actions (Links zu Unterseiten)"
            options="{
                icon: '{f:uri.resource(path: \'Icons/Content/CallToActionBoxes.svg\')}',
                group: 'Globale Elemente',
                Fluidcontent: {
                    sorting: 200
                }
            }"
    >
        <flux:form.container name="cta.first" label="Erste Box">
            <flux:field.input name="title" label="Titel" required="1" />
        </flux:form.container>
        <flux:form.container name="cta.second" label="Zweite Box">
            <flux:field.input name="title" label="Titel" required="1" />
        </flux:form.container>
        <flux:form.container name="cta.third" label="Dritte Box">
            <flux:field.input name="title" label="Titel" required="1" />
        </flux:form.container>
    </flux:form>
</f:section>
```
